src/components/CreateVmWizard/steps/BasicSettings.js

```diff
@@ -547,6 +594,48 @@ class BasicSettings extends React.Component {
             </FieldRow>
           </React.Fragment>
         }
+
+        {/* Advanced options */}
```

Comment:
Well, I was thinking about improving displaying the _cloud-init/sysprep_ and _Start on creation_ checkboxes, too, in a separate PR. However, I haven't received clear info from the UX regarding this, just some assumptions, possibilities.

IMHO, it makes sense to place those checkboxes in the _Advanced Options_ expand section, too, because the actual position of those checkboxes isn't optimal - they make a feel that those options are closely connected and related to the _Optimized for_ drop down, and because I think those are advanced options, too. And also because I can see that _Cloud-init_ can be enabled specifically in the _Advanced Options_ of a VM Details card.

Yes, there's a possibility to rename the section to "Advanced CPU Topology" as you suggest. However, with renaming it, I think it will be less consistent with the same section containing CPU topology options in the VM _Details_ card, where I was inspired.

So I would choose the first option to solve this thing- to place those checkboxes into the _Advanced options_ section.

Reply:
It does make sense to visually group the 3 advanced things together (topology, start on create, cloud-init/sysprep).

However, I'd rather not put the start-on-create and cloud-init/sysprep options inside a click-to-see advanced section.  I'd rather see those two all the time, and then force a click to see the topology.

Let's limit scope creep.  The easiest thing to do right now is not worry about start-on-create and cloud-init/sysprep simply because they're not part of the issue this PR is solving.  We can raise a new issue and discuss if we actually want to do that later.  Something like "On the create VM wizard, consider grouping ____ in a default collapsed advanced options section".  We can then have a short meeting with people just to discuss if that is something we want to do or not.

Reply:
Of course I didn't want to move `cloud-init/sysprep` and `Start on creation` options anywhere, within this PR :)

Reply:
>>  In VM details the Advanced Options section covers also cloud-init.
>>  How about including both Start virtual machine on creation and Enable Cloud-init/Sysprep options in the wizard 
>>  version?
> Could you please, explain this suggestion more? Thanks! :)

I do agree that we should stop feature creep :) But IMHO both checkboxes belong to advance section.

Reply:
> I do agree that we should stop feature creep :) But IMHO both checkboxes belong to advance section.

Great!  It'll be done with https://github.com/oVirt/ovirt-web-ui/issues/1297.  This PR's content is full.